### PR TITLE
Aro quay docs

### PR DESCRIPTION
**Pull request by kwhitemobb**

### CODE REVIEW

File: docs/aro/setup-quay/quay-console.md [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+## Video Walkthrough
+
+If you prefer a more visual medium, you can watch [Kristopher White] walk through Quay Registry Storage Setup on [YouTube]([https://youtu.be/iifsB-uuEFc](https://youtu.be/yMmSrx4hN70)).
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/yMmSrx4hN70" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+
+## Prerequisites
+
+### Azure CLI
```
Comment (paulczar) [inline docs/aro/setup-quay/quay-console.md:17]: no need for these instructions, refer back to ARO quickstart guide, or the CLI install docs.

File: docs/aro/setup-quay/quay-console.md [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+1. Register resource providers
+
+    ```bash
+    az provider register -n Microsoft.RedHatOpenShift --wait
+    az provider register -n Microsoft.Compute --wait
+    az provider register -n Microsoft.Storage --wait
+    az provider register -n Microsoft.Authorization --wait
+    ```
+
+### Get Red Hat pull secret
```
Comment (paulczar) [inline docs/aro/setup-quay/quay-console.md:86]: no need for these instructions, refer back to ARO quickstart guide.

File: docs/aro/setup-quay/quay-console.md [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+
+1. Use the URL and the credentials provided by the output of the last two commands to log into OpenShift via a web browser.
+
+![ARO login page](./images/aro-login.png)
+
+## Red Hat Quay Setup
+
+### Backend Storage Setup
+
+1. **Login** to Azure
```
Comment (paulczar) [inline docs/aro/setup-quay/quay-console.md:210]: provide a link to portal.azure.com or whatever the base URL is for azure cloud console..

**Review by paulczar — CHANGES_REQUESTED** [review]

**Comment by kwhitemobb** [comment]
done

File: docs/aro/setup-quay/quay-cli.md [hunk 4]
```diff
@@ -0,0 +1,117 @@
+# Setting up Quay on an ARO cluster using Azure Container Storage 
+
+**Kristopher White x Connor Wooley**
+
+*07/25/2022*
+
+## Pre Requisites
+
+* an ARO cluster
```
Comment (paulczar) [inline docs/aro/setup-quay/quay-cli.md:9]: link to the ARO quickstart.   `[An ARO cluster](/docs/quickstart-aro.html)`

File: docs/aro/setup-quay/quay-cli.md [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+* oc/azure cli
```
Comment (paulczar) [inline docs/aro/setup-quay/quay-cli.md:10]: separate lines, they're two different CLIs,  link to the download or install instructions for each.

File: docs/aro/setup-quay/quay-console.md [hunk 6]
```diff
@@ -0,0 +1,103 @@
...
+
+If you prefer a more visual medium, you can watch [Kristopher White] walk through Quay Registry Storage Setup on [YouTube]([https://youtu.be/iifsB-uuEFc](https://youtu.be/yMmSrx4hN70)).
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/yMmSrx4hN70" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## Red Hat Quay Setup
+
+### Backend Storage Setup
+
+1. **Login** to Azure https://portal.azure.com/
```
Comment (paulczar) [inline docs/aro/setup-quay/quay-console.md:18]: **Login** to `[Azure](https://portal.azure.com/)`

**Review by paulczar — CHANGES_REQUESTED** [review]

File: docs/aro/setup-quay/quay-cli.md (hunk 4 above)
Comment (kwhitemobb) [reply docs/aro/setup-quay/quay-cli.md:9]: deleted it already

**Review by paulczar — APPROVED** [review]

**Pull request merged**